### PR TITLE
Dockerfile

### PR DESCRIPTION
**Pull request by doismellburning**
Add Dockerfile and basic GitHub Actions testing

This goes part way to enabling more convenient local dev, rather than requiring everything to be done on `pip.srcf` - it thus also goes part way to enabling automated testing (via GitHub Actions)

There is still plenty to be done here - e.g. an actual test that the service starts up and runs and does proper things - but this does a fair bit of the initial work as well as enforcing that all dependencies are present and explicitly declared.

### CODE REVIEW

**Comment by doismellburning** [comment]
(You might need to go via https://github.com/doismellburning/control-panel/actions?query=workflow%3ADocker to see the Actions outcome)

**Comment by doismellburning** [comment]
Where can I find a packaged `ucam_webauth`?

**Comment by doismellburning** [comment]
Also where's the best place for me to get `srcf.controllib.job_runner` and `wsgi-proxy-run` from? (cf. the README) - I see they're installed on `pip.srcf` but are they distributed in any fashion / how to best install them?

**Comment by doismellburning** [comment]
Thanks @rsa33 for pointing me at https://launchpad.net/~malcscott/+archive/ubuntu/python-ucam-webauth !

**Comment by doismellburning** [comment]
Hm, `python-ucam-webauth` is a 2.7 package and the README suggests this is Python 3 software. Has anyone packaged a `python3-ucam-webauth` or is the copy on `pip.srcf` installed with pip perhaps?

**Comment by doismellburning** [comment]
...`python3-ucam-webauth` was also in the PPA, I was looking at source packages not binary packages, my bad!

**Comment by doismellburning** [comment]
I've pulled out the most interesting bits of my original checklist into assorted issues:

* #5 
* https://github.com/SRCF/srcf-python/issues/1
* https://github.com/SRCF/SRCFLib/issues/2

However I think this is reasonable to merge as-is in its current form - yes the Docker setup is half-baked but it still adds *some* basic automated testing and is a good foundation for more

File: README.md [hunk 1]
```diff
@@ -69,3 +69,17 @@ On pip:
 sudo update-usr-local
 sudo systemctl restart control-panel-job-runner.service
 ```
+
+# Docker
+
+There is a basic `Dockerfile` to ensure dependencies are tracked and to assist (some forms) of testing and local development.
```
Comment (edwinbalani) [inline README.md:75]: the closing ')' probably wants to be a word later, but this is barely a nit

**Review by edwinbalani — APPROVED** [review]

**Pull request merged**